contracts/RewardManager.sol

```diff
@@ -103,6 +109,7 @@ contract RewardManager is Ownable, Versionable, Safe {
     rewardeeRegistrationFeeInSPEND = _rewardeeRegistrationFeeInSPEND;
     rewardProgramRegistrationFeeInSPEND = _rewardProgramRegistrationFeeInSPEND;
     versionManager = _versionManager;
+    governanceAdmin = _governanceAdmin;
```

Comment:
Do we need a setter so we can update this?

Reply:
We’ve been saving on contract size by consolidating the setters into this general setup() function